deepfence_agent/supervisord-cloud.conf

```diff
@@ -0,0 +1,25 @@
+[unix_http_server]
+file=/var/run/supervisor.sock
+chmod=0700
+
+[supervisord]
+logfile=%(ENV_DF_INSTALL_DIR)s/var/log/supervisor/supervisord.log
+pidfile=/var/run/supervisord.pid
+childlogdir=%(ENV_DF_INSTALL_DIR)s/var/log/supervisor
+
+[supervisorctl]
+serverurl=unix:///var/run/supervisor.sock
+
+[rpcinterface:supervisor]
+supervisor.rpcinterface_factory = supervisor.rpcinterface:make_main_rpcinterface
+
+[program:deepfenced]
+environment=DF_ENABLE_CLOUD_NODE=true
+command=/bin/deepfenced
+numprocs=1
+autostart=true
+autorestart=true
+startsecs=0
+stopwaitsecs=30
+killasgroup=true
+priority=50
```

Comment:
Can't we resue the same as agent?

Reply:
We are using the DF_INSTALL_DIR env variable for the cloud scanner agent which we are not using in the case of regular agent.  This env is required to setup the directory structure (/home/deepfence) for the cloudscanner agent.